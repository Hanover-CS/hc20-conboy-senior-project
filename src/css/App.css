.App {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  justify-content: top;
  font-size: calc(10px + 2vmin);
  color: white;
}