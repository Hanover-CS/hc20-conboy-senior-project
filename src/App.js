/* Author: Patrick Conboy

 * Date Created: 11/1/2019

 * Description: This file just pulls together the various main web page components
 * of our project.
 */

import React, { Component } from 'react'
import SearchPage from './SearchPage'
import './App.css'

class App extends Component {
  // Constructor currently does nothing, but
  // might need later for login functionality.
  constructor() {
    super()
  }

  render() {
    return (
      <div className="App">
        <SearchPage />
      </div>
    )
  }
}

export default App;
