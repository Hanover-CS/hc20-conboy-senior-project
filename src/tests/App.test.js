import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import App from '../App'

describe('App Tests', () => {
  test('app renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<App />, div)
    ReactDOM.unmountComponentAtNode(div)
  })


  test('snapshot of app hasn\'t changed', () => {
    const component = renderer.create(<App />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})