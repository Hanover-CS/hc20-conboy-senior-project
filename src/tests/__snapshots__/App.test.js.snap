// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Tests snapshot of app hasn't changed 1`] = `
<div
  className="App"
>
  <div
    className="NavBar"
  >
    <ul
      id="navLinks"
    >
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Home
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/searchpage"
          onClick={[Function]}
        >
          Search Page
        </a>
      </li>
    </ul>
  </div>
</div>
`;
